src/stories/MovieCard.vue

```diff
@@ -0,0 +1,55 @@
+<script setup lang="ts">
+import { defineProps } from 'vue';
+
+interface Movie {
+    name: string;
+    genre: string;
+    year: number;
+    rating: number;
+    movieLength: number;
+    description: string;
+    poster: string;
+}
+defineProps<{ movie: Movie }>()
+</script>
+
+<template>
+    <div class="card">
+        <img 
+        class="card__img" :src="`${movie.poster}`" :alt="movie.name" />
+        <div class="card__info">
+            <div class="card__info__name">{{ movie.name }}</div>
```

Comment:
You can't use elment in another element by class, better to use the `info` as block:

```suggestion
            <div class="card-info__name">{{ movie.name }}</div>
```

Reply:
Done